Events/hirse-024-2024-01.md

```diff
@@ -0,0 +1,37 @@
+# Webinar: Best practice made easy: Deploying tools for FAIR research software development
+
+- Event Date: 2024-01-23
+- Location: Online
+- Event Website: <https://www.helmholtz-hirse.de/series/2024_01_23-seminar_24.html>
+- Organizers: Helmholtz Platform for Research Software Engineering
+
+This event is part of the HiRSE Seminar Series, a series of talks revolving around topics relevant to research software engineering and research software engineers. The seminars cover a broad range of topics from political to technical ones, and take place every 2 to 4 weeks.
+
+Resource Information | Details
+:--- | :---			   
+Webinar Title | [Best practice made easy: Deploying tools for FAIR research software development](https://www.helmholtz-hirse.de/series/2024_01_23-seminar_24.html)
+Date and Time | 2024-01-23 11:00am-12:00pm CET
+Presenter | Anton Pirogov (Forschungszentrum Jülich)
+Presentation Language | English
+Request Connection Information | [hirse@fz-juelich.de](mailto:hirse@fz-juelich.de)
+Series Information | <https://www.helmholtz-hirse.de/series.html>
+Archives | [Recordings](https://www.youtube.com/@Helmholtz_Platform_for_RSE) (YouTube), [Slides](https://zenodo.org/communities/hirse/) (Zenodo)
+
+### Abstract
+
+Sustainable software development and metadata practices are crucial for making research software FAIR. Undeniably, this requires an initial investment of time and effort for researching and adopting best practices, as well as for regular maintenance tasks. This impairs the bottom-up adoption of best practices by RSEs. In this talk we present two complementary tools addressing this challenge:
+
+The command-line tool [somesy](https://pypi.org/project/somesy) organizes and synchronizes software project metadata across multiple required or recommended files that are typically used in a software project. It is designed for easy integration into the development workflow, reducing the overhead for software metadata management.
+
+The [fair-python-cookiecutter](https://github.com/Materials-Data-Science-and-Informatics/fair-python-cookiecutter) is a git repository template providing a well-structured foundation for new Python projects. The detailed documentation also turns it into a hands-on educational resource. Besides combining various state-of-the-art tools, it includes somesy to simplify software metadata management. The template follows software engineering practices recommended by DLR and OpenSSF. Furthermore, it supports relevant metadata standards such as REUSE, CITATION.cff and CodeMeta.
+
+### Presenter
+
+- Anton Pirogov (Forschungszentrum Jülich)
+
+#### Contributed by [Claire Wyatt](https://github.com/ClaireWyatt)
```

Comment:
If that's the place where the authors are taken from, then listing Anton here might be a good idea, too (as suggested): https://github.com/apirogov